data/coalition missions.txt

```diff
@@ -60,12 +60,19 @@ mission "Coalition: First Contact"
 			label oppressive
 			`	The spider says, "In separate boxes, they each species keep. And in subtle ways, your development and discoveries they limit. And on your technology, restrictions they place."`
 			`	The beetle says, "Or, wrong are we? And the jump drive that brought you here, a gift from the Quarg it is? They its secret no longer hoard?" You tell it that the Quarg do indeed refuse to give jump drives to humans, and it says, "Then, prisoners indeed you are."`
-			label next
+			branch next
+				has "First Contact: Hai: offered"
 			choice
 				`	"I suppose you may be right about the Quarg."`
 					goto end
 				`	"But, the Quarg do let the humans and the Hai make contact with each other."`
 			`	This, apparently, is news to them. You explain to them who the Hai are, and that some humans have made contact with them and are living in their territory. "A second Coalition you are forming!" says the centaur, tapping its hooves on the floor in excitement. "Once again, foiled the Quarg are. A secret way for species to work together, they have overlooked. Great things, from this may come."`
+					goto end
+			label next
+			choice
+				`	"I suppose you may be right about the Quarg."`
```

Comment:
@jafdy, that _was_ intended, but thanks for pointing out the weirdness in my sentence. Fixed.